statsmodels/graphics/tests/test_gofplots.py

```diff
@@ -98,6 +103,10 @@ def test_ppplot_other_prbplt(self, close_figures):
 
     @pytest.mark.xfail(strict=True)
     @pytest.mark.matplotlib
+    @pytest.mark.skipif(
+        PYTHON_IMPL_WASM,
+        reason="Matplotlib uses different backend in WASM/Pyodide"
```

Comment:
Does the backend matter for these and the others that are marked to skip?

Reply:
If you mean the functionality of the plots, I don't think so. Normally, one will not need to adapt the code to use `matplotlib-pyodide` to use the HTML5/WASM canvases, since it gets installed when one installs `matplotlib` using `micropip`. The details are a little murky since it has been some time for me, but I received the error: `'Line2D' object has no attribute "other"`, so it is probably a missing feature in `matplotlib-pyodide` at the time of writing.

For example, https://github.com/statsmodels/statsmodels/blob/main/examples/python/categorical_interaction_plot.py (I can't find any other failing example right now) can be tried out in the Pyodide console (https://pyodide.org/en/stable/console.html), though it won't display any of the `matplotlib.figure.Figure` objects, of course, lacking a canvas for the renderer – they can be viewed in a JupyterLite notebook (https://jupyter.org/try-jupyter/lab/) and they should all work without any code changes. Not all of the tests marked with `@pytest.mark.matplotlib` have been skipped, and the ones skipped here are those that were failing because of the error noted above. I could investigate further as a follow-up if you would want me to do so.